lumigator/python/mzai/backend/backend/api/routes/jobs.py

```diff
@@ -0,0 +1,61 @@
+from uuid import UUID
+
+from fastapi import APIRouter, status
+from schemas.extras import ListingResponse
+from schemas.jobs import (
+    JobCreate,
+    JobResponse,
+    JobResultDownloadResponse,
+    JobResultResponse,
```

Comment:
Just a note for later on that we are not currently using Job results as everything's stored in the output files and we might want to remove this when we move to proper experiment tracking / artifact storage.

Reply:
should I take this out in that case, wdyt?

Reply:
I think this is out of this PR's scope, but I'll add an issue for that